File: Leetcode/2873 - Maximum Value of an Ordered Triplet I/solution.js
var maximumTripletValue = function(nums) {
    let maxTripletScore = 0;
    for (let i = 0; i < nums.length; i++) {
        for (let j = i+1; j < nums.length; j++) {
            for (let k = j+1; k < nums.length; k++) {
                let tripletScore = (nums[i] - nums[j]) * nums[k];
                maxTripletScore = Math.max(maxTripletScore, tripletScore);
            }
        }
    }
    return maxTripletScore;
};

File: Leetcode/2873 - Maximum Value of an Ordered Triplet I/solution.py
def maximumTripletValue(self, nums):
    max_triplet_score = 0
    for i in range(len(nums)):
        for j in range(i+1, len(nums)):
            for k in range(j+1, len(nums)):
                triplet_score = (nums[i] - nums[j]) * nums[k]
                max_triplet_score = max(max_triplet_score, triplet_score)
    return max_triplet_score